example_config.toml

```diff
@@ -96,6 +95,7 @@
 
     ## IBM Quantum Experience authentication token
     # ibmqx_token = "XXX"
+    # shots = 1000
```

Comment:
We need to be careful here: hardware devices now have to be initialised with finite shot numbers.

Reply:
What happens if you don't initialize with a set number of shots? An error?

I'm in favour of having a default.

Reply:
I am also in favour of a default, but this is defined in the devices themselves. In some sense, the toml file is just an example of how to consistently change the default for a user.